### PR TITLE
Wlt 951 move depositmigration

### PR DESCRIPTION
**Pull request by sansaian**
**- What I did**
move method deposit migration from member contract to migrationdaemon contract
**- How I did it**
according to the scheme https://insolar.atlassian.net/wiki/spaces/DEV/pages/833257510/WIP+Migration+Admin+Daemon
**- How to verify it**
start function test
**- Description for the changelog**

### CODE REVIEW

File: logicrunner/builtin/foundation/helpers.go [hunk 1]
```diff
@@ -98,6 +94,23 @@ func IsMigrationDaemonMember(member insolar.Reference) bool {
 	return false
 }
 
+func GetMigrationDaemonByMemberRef(memberRef string) (insolar.Reference, error) {
```
Comment (LinnikD) [inline logicrunner/builtin/foundation/helpers.go]: Do we really need both GetMigrationDaemonByMemberRef and GetMigrationDaemon in foundation. Its better to keep foundation small

File: logicrunner/builtin/contract/migrationdaemon/migrationdaemon.go [hunk 2]
```diff
@@ -17,17 +17,54 @@
...
 	IsActive              bool
 	MigrationDaemonMember insolar.Reference
 }
 
+// Migration methods.
+type DepositMigrationResult struct {
+	Reference string `json:"memberReference"`
+}
+
+//Call internal function migration admin from api.
```
Comment (LinnikD) [inline logicrunner/builtin/contract/migrationdaemon/migrationdaemon.go]: Codestyle: comment on public var should started with name like:  
// DepositMigrationCall calls internal function migration admin from api.

File: logicrunner/builtin/contract/migrationdaemon/migrationdaemon.go [hunk 3]
```diff
@@ -17,17 +17,54 @@
...
+const CONVERSION = "10"
+
 // MigrationDaemonContract make migration procedure.
 type MigrationDaemon struct {
 	foundation.BaseContract
 	IsActive              bool
 	MigrationDaemonMember insolar.Reference
 }
 
+// Migration methods.
```
Comment (LinnikD) [inline logicrunner/builtin/contract/migrationdaemon/migrationdaemon.go]: Codestyle

File: insolar/genesisrefs/refs.go [hunk 4]
```diff
@@ -99,10 +99,11 @@ var (
 		return
 	}()
 
-	// ContractMigrationDaemons is the migration daemon contracts references,which is associated with MigrationDaemonMember.
-	ContractMigrationDaemons = func() (result [insolar.GenesisAmountMigrationDaemonMembers]insolar.Reference) {
-		for i, name := range insolar.GenesisNameMigrationDaemons {
-			result[i] = GenesisRef(name)
+	//ContractMigrationMap where key is migration daemon member  references and value related migration daemon contract
```
Comment (LinnikD) [inline insolar/genesisrefs/refs.go]: // ContractMigrationMap

File: logicrunner/builtin/foundation/helpers.go (hunk 1 above)
Comment (sansaian) [reply logicrunner/builtin/foundation/helpers.go]: Done. Move to migration admin

File: logicrunner/builtin/contract/migrationdaemon/migrationdaemon.go (hunk 2 above)
Comment (sansaian) [reply logicrunner/builtin/contract/migrationdaemon/migrationdaemon.go]: Fix

File: logicrunner/builtin/contract/migrationdaemon/migrationdaemon.go (hunk 3 above)
Comment (sansaian) [reply logicrunner/builtin/contract/migrationdaemon/migrationdaemon.go]: Fix

File: insolar/genesisrefs/refs.go (hunk 4 above)
Comment (sansaian) [reply insolar/genesisrefs/refs.go]: Fix

**Pull request merged**